pymbar/mbar_solvers.py

```diff
@@ -253,12 +284,11 @@ def adaptive(u_kn, N_k, f_k, tol=1.0e-12, options=None):
 
     options: dictionary of options
         gamma (float between 0 and 1) - incrementor for NR iterations (default 1.0).  Usually not changed now, since adaptively switch.
-        maximum_iterations (int) - maximum number of Newton-Raphson iterations (default 250: either NR converges or doesn't, pretty quickly)
+        maxiter (int) - maximum number of Newton-Raphson iterations (default 10000: either NR converges or doesn't, pretty quickly)
```

Comment:
Why change the name from something clear and comprehensible (`maximum_iterations`) to something shorter (`maxiter`)? Why change the name at all?